### DISCUSSION TITLE
Soundfile that plays on "OnUse" doesn't seem to work on Multiplayer, but works in Singleplayer.

### DISCUSSION THREAD

**Discussion by MrWomanStealer** [FakeFishGames/Barotrauma#14412] [Bug reports]
### Disclaimers

- [X] My issue happened while using mods.
- [X] I have searched the issue tracker to check if the issue has already been reported.

### What happened?

Trying to mod in an item that when placed, plays a sound.
Currently, it plays the sound perfectly fine in Singleplayer, but when in a multiplayer environment, it simply doesn't play a sound.
No console errors.

Currently, I am using this XML.
```xml
    <Holdable selectkey="Select" pickkey="Use" slots="Any,RightHand,LeftHand" msg="ItemMsgDetach" PickingTime="1" aimpos="85,-10" handle1="0,0" attachable="true" aimable="true">
      <Sound file="%ModDir%/Content/sounds/vaporeoncopypasta.ogg" type="OnUse" range="500"/>
    </Holdable
```
I've attached a singleplayer and multiplayer video.

https://github.com/user-attachments/assets/7b83d1fc-3ef4-4603-9d5e-8b3db7ca0614

https://github.com/user-attachments/assets/89dbea2c-8fc3-4be2-b3d7-189c7bef1183

### Reproduction steps

1. Start multiplayer server
2. Spawn in item
3. Place
4. Expect sound, get nothing

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by aoyamajia** [comment]
[Here's a less degenerate minimal reproducible example zipped as a mod](https://github.com/user-attachments/files/16539904/mre.zip)
Find the item/mod as "attachable holdable," this example also only plays in singleplayer. It should play GuitarClown.ogg upon attaching with `E`.

**Comment by Jasontti** [comment]
Thank you for the report.
I was able to replicate the problem with the mod provided. Turning this into an issue.

**Comment by itchyOwl** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/pull/5638
